client/app/components/ParameterValueInput.jsx

```diff
@@ -103,7 +104,12 @@ class ParameterValueInput extends React.Component {
         mode={parameter.multiValuesOptions ? "multiple" : "default"}
         value={normalize(value)}
         onChange={this.onSelect}
-        options={map(enumOptionsArray, opt => ({ label: String(opt), value: opt }))}
+        options={enumOptionsArray.map((option) => {
```

Comment:
Since this file is still in JavaScript (and a good part of the code too), it's better to leave the `lodash` `map` for now, just to be safe 😅